ckan.ini

```diff
@@ -0,0 +1,338 @@
+##    With debug mode enabled, a visitor to your site could execute malicious
+##    commands.
+debug = false
+
+[app:main]
+
+## General settings ############################################################
+use = egg:ckan
+ckan.legacy_route_mappings = {}
+config.mode = default
+
+## Development settings ########################################################
+ckan.devserver.host = localhost
+ckan.devserver.port = 5000
+ckan.devserver.threaded = false
+ckan.devserver.multiprocess = 1
+ckan.devserver.watch_patterns = 
+ckan.devserver.ssl_cert = 
+ckan.devserver.ssl_key = 
+
+## Session settings ############################################################
+ckan.user.last_active_interval = 600
+cache_dir = /tmp/%(ckan.site_id)s
+beaker.session.key = ckan
+beaker.session.secret = 9EZiPwkeS+cZpqb0VDWrN+Q0M
+beaker.session.auto = false
+beaker.session.cookie_expires = false
+# beaker.session.cookie_domain = .example.com
+beaker.session.save_accessed_time = true
+beaker.session.secure = false
+beaker.session.timeout = 600
+
+## Database settings ###########################################################
+sqlalchemy.url = postgresql://ckan:ckan@db/ckan
+sqlalchemy.pool_pre_ping = true
+# sqlalchemy.<OPTION> = 
+
+## Site Settings ###############################################################
+ckan.site_url = http://localhost:5003
+apikey_header_name = X-CKAN-API-Key
+ckan.cache_expires = 0
+ckan.cache_enabled = false
+ckan.mimetype_guess = file_ext
+ckan.static_max_age = 3600
+ckan.tracking_enabled = false
+ckan.valid_url_schemes = http https ftp
+ckan.requests.timeout = 5
+ckan.hide_version = false
+ckan.redirect_to_login_if_not_authorized = true
+
+## Authorization Settings ######################################################
+ckan.auth.anon_create_dataset = false
+ckan.auth.create_unowned_dataset = false
+ckan.auth.create_dataset_if_not_in_organization = false
+ckan.auth.user_create_groups = false
+ckan.auth.user_create_organizations = true
+ckan.auth.user_delete_groups = false
+ckan.auth.user_delete_organizations = true
+ckan.auth.create_user_via_api = true
+ckan.auth.create_user_via_web = true
+ckan.auth.roles_that_cascade_to_sub_groups = admin
+ckan.auth.public_user_details = true
+ckan.auth.public_activity_stream_detail = false
+ckan.auth.allow_dataset_collaborators = false
+ckan.auth.allow_admin_collaborators = false
+ckan.auth.allow_collaborators_to_change_owner_org = false
+ckan.auth.create_default_api_keys = false
+ckan.auth.login_view = user.login
+ckan.auth.reveal_private_datasets = false
+ckan.auth.enable_cookie_auth_in_api = true
+ckan.auth.route_after_login = dashboard.datasets
+
+## User Account Creation Setting
+; ckan.valid_email_regexes = .gov.uk$ .nhs.uk$ .nhs.net$ .ac.uk$ .os.uk$ .mod.uk$ .police.uk$ .bl.uk$
+
+## CSRF Protection #############################################################
+WTF_CSRF_ENABLED = true
+WTF_CSRF_CHECK_DEFAULT = true
+WTF_CSRF_SECRET_KEY = _e6qkoYldyll9U3zfy5n6rg-JxCPY4DUfy5LHu3qnUo
```

Comment:
Is this used for anything real? If so you probably need to change it 😅

Reply:
Not used for anything real, just auto generated so should be ok for local dev use, I'll make sure that the ckan chart auto generates when deployed